mkContainer/mkContainer.nix

```diff
@@ -96,6 +95,41 @@ let
     ];
   };
 
+  # For field definitions, see `ActivateCtx` in `flox-core`
+  activateCtx = {
+    mode = "${activationMode}";
+    shell = {
+      Bash = "${containerPkgs.bashInteractive}/bin/bash";
+    };
+    env = "${environment}";
+    run_args = [ ];
+    invocation_type = null;
+    remove_after_reading = false;
+    env_description = "${containerName}";
+    env_cache = "/tmp";
+    flox_env_log_dir = "/log/flox";
```

Comment:
question blocking: should we create `/log/flox`?
nonblocking suggestion: I think FHS says to use `/var/log`?
question nonblocking: I think this is a behavior change, but for the better - seems like the watchdog may be getting passed an empty string for log dir currently? And somehow it's handling it gracefully? Is that your understanding?

Reply:
Wait, are we even starting the watchdog in the container? I don't think so, in which case, that log directory isn't even really getting used.